docs/content/docs/dev/table/sql/queries/hints.md

```diff
@@ -84,4 +84,244 @@ insert into kafka_table1 /*+ OPTIONS('sink.partitioner'='round-robin') */ select
+-- For this case, default join strategy will be adopted instead of BroadCast join.
+SELECT /*+ BROADCAST(t1) */ * FROM t1 join t2 ON t1.id > t2.id;
+
+-- BROADCAST Join Hint strategy don't support full outer join.
+-- For this case, default join strategy will be adopted instead of BroadCast join.
+SELECT /*+ BROADCAST(t1) */ * FROM t1 FULL OUTER JOIN t2 ON t1.id = t2.id;
+```
+
+#### SHUFFLE_HASH
+
+{{< label Batch >}}
+
+For this type of `Join Hint`, suggests that join use `Shuffle Hash join` strategy. Compare of `BROADCAST`, This type of
+`Joint Hint` is suitable for the scenario of the data volume of both table is not small enough.
+
+*NOTE:* `SHUFFLE_HASH Join Hint` only supports join with equivalence join condition.
+
+##### Examples
+
+```sql
+CREATE TABLE t1 (id BIGINT, name STRING, age INT) WITH (...);
+CREATE TABLE t2 (id BIGINT, name STRING, age INT) WITH (...);
+CREATE TABLE t3 (id BIGINT, name STRING, age INT) WITH (...);
+
+-- Select t1 as join side with this hint, t1 will be the join build side.
+SELECT /*+ SHUFFLE_HASH(t1) */ * FROM t1 JOIN t2 ON t1.id = t2.id;
+
+-- Select t1 and t3 as join side with this hint.
+-- For this case, when join between t1 and t2, t1 will be the build side,
+-- And when join between the result after t1 joins t2 and t3, t3 will be the build side.
+SELECT /*+ SHUFFLE_HASH(t1, t3) */ * FROM t1 JOIN t2 ON t1.id = t2.id JOIN t3 ON t1.id = t3.id;
+
+-- SHUFFLE_HASH Join Hint don't support non-equivalent join conditions.
+-- For this case, default join strategy will be adopted instead of Hash Shuffle join.
+SELECT /*+ SHUFFLE_HASH(t1) */ * FROM t1 join t2 ON t1.id > t2.id;
+```
+
+#### SHUFFLE_MERGE
+
+{{< label Batch >}}
+
+For this type of `Join Hint`, suggests that join use `Sort Merge join` strategy. This type of `Join Hint` is recommended for
+using in the scenario of joining between two large tables or the scenario that the data at both sides of the join is already in order.
+
+*NOTE:* `SHUFFLE_MERGE Join Hint` strategy only supports join with equivalence join condition.
+
+##### Examples
+
+```sql
+CREATE TABLE t1 (id BIGINT, name STRING, age INT) WITH (...);
+CREATE TABLE t2 (id BIGINT, name STRING, age INT) WITH (...);
+CREATE TABLE t3 (id BIGINT, name STRING, age INT) WITH (...);
+
+-- Sort Merge join strategy is adopted.
+SELECT /*+ SHUFFLE_MERGE(t1) */ * FROM t1 JOIN t2 ON t1.id = t2.id;
+
+-- Sort Merge join strategy is both adopted in these two joins.
+SELECT /*+ SHUFFLE_MERGE(t1, t3) */ * FROM t1 JOIN t2 ON t1.id = t2.id JOIN t3 ON t1.id = t3.id;
+
+-- SHUFFLE_MERGE Join Hint don't support non-equivalent join conditions.
+-- For this case, default join strategy will be adopted instead of Sort Merge join.
+SELECT /*+ SHUFFLE_MERGE(t1) */ * FROM t1 join t2 ON t1.id > t2.id;
+```
+
+#### NEST_LOOP
+
+{{< label Batch >}}
+
+For this type of `Join Hint`, suggests that join use `Nested Loop join` strategy. This type of join hint is not recommended 
```

Comment:
Suggests that Flink uses nested loop join.

Reply:
> Suggests that Flink uses nested loop join.

Done!